requirements-igwn.txt

```
@@ -1,6 +1,6 @@
 # For LDG service access
-ciecplib[kerberos]>=0.7.0
-dqsegdb2>=1.0.1
+ciecplib[kerberos] >= 0.7.0
+dqsegdb2 >= 1.1.1
```

1.1.4 is the latest bug fix release for 1.1 that includes a fix for automatically discovering tokens. 

```suggestion
dqsegdb2 >= 1.1.4
```